tests/dialects/test_clickhouse.py

```diff
@@ -398,10 +396,6 @@ def test_clickhouse(self):
                 "clickhouse": "SELECT quantileIf(0.5)(a, TRUE)",
             },
         )
-        self.validate_all(
-            "SELECT position(needle IN haystack)",
-            write={"clickhouse": "SELECT position(haystack, needle)"},
-        )
```

Comment:
Ditto, while we're at it let's also change `validate_all` into `validate_identity(in, out)`.